rally-domain/src/main/groovy/yakworks/rally/orgs/repo/ContactRepo.groovy

```diff
@@ -64,14 +64,7 @@ class ContactRepo extends LongIdGormRepo<Contact> {
         //remove
         TagLink.remove(contact)
 
-        // XXX why are we keeping the locations around?
-        // if its a location for a contact it should be deleted along with the contact right?
-        Location.executeUpdate("update Location set contact = null where contact = :contact", [contact: contact]) //set contact to null
```

Comment:
@jdabal this was done like this another CED hack. can you check into it. was some kind of special contact for bill to or something that had the location assigned. I looked into some months back